recipe/meta.yaml

```
@@ -32,13 +41,26 @@ requirements:
     - numpy 1.23.*  # [py==311]
     - numpy 1.26.*  # [py>=312]
     - python-build
+    - libabseil 20240116.2
```

```suggestion
    - libabseil {{ libabseil }}
```
And we should do the same with all deps defined in our aggregate CBC file.